hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -4779,8 +4788,8 @@ public boolean checkAndRowMutate(byte[] row, Filter filter, TimeRange timeRange,
     return checkAndMutate(checkAndMutate).isSuccess();
   }
 
-  @Override
-  public CheckAndMutateResult checkAndMutate(CheckAndMutate checkAndMutate) throws IOException {
+  private CheckAndMutateResult checkAndMutateInternal(CheckAndMutate checkAndMutate)
```

Comment:
Why modify this to a private method?

Reply:
Sorry. got it.

Reply:
Haha, to better suit the TraceUtil.trace call :)